moya/doclinker.ts

```diff
@@ -82,7 +82,7 @@ const commentFooter = `
 
 const isIssueQuestion = (): boolean => {
   const labels = danger.github.issue.labels
-  return labels.map(i => i.name).includes('question')
+  return labels.filter(i => i.name == "question").length > 0
```

Comment:
`labels.some(i =< i.name === "question")` -  

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Beautiful 👌 Thank you!

Reply:
@orta Were you suggesting this as sugar or is my logic wrong? It seems that peril is commenting on any labeled event. `danger.github.issue.labels` should only contain labels that exist on the issue and not all possible labels for the repo, correct?